package.json

```diff
@@ -98,93 +100,94 @@
     "javascript-state-machine": "^3.1.0",
-    "json-canonicalize": "^1.0.4",
+    "json-canonicalize": "^1.0.6",
     "lodash": "^4.17.21",
-    "module-alias": "^2.2.2",
-    "openapi-backend": "^5.3.0",
-    "openapi-jsonschema-parameters": "^12.0.0",
+    "module-alias": "^2.2.3",
+    "openapi-backend": "^5.9.2",
+    "openapi-jsonschema-parameters": "^12.1.3",
     "path": "^0.12.7",
     "promise-timeout": "^1.3.0",
-    "random-word-slugs": "^0.1.6",
+    "random-word-slugs": "^0.1.7",
     "redis": "3.1.2",
-    "typescript": "4.7.4",
-    "uuid": "^8.3.2",
-    "ws": "^8.8.0"
+    "typescript": "5.2.2",
+    "uuid": "^9.0.1",
+    "ws": "^8.14.2"
   },
   "@comment devDependencies": [
     "Caution advised in upgrading redis-mock past 0.52.0. Investigation needed."
   ],
   "devDependencies": {
-    "@commitlint/cli": "^17.0.2",
-    "@commitlint/config-conventional": "^17.0.2",
+    "@commitlint/cli": "^17.7.1",
+    "@commitlint/config-conventional": "^17.7.0",
     "@immobiliarelabs/standard-version": "^10.1.0",
-    "@redocly/cli": "^1.0.0-beta.102",
-    "@types/convict": "^6.1.1",
-    "@types/crypto-js": "^4.1.1",
-    "@types/hapi": "^18.0.7",
-    "@types/hapi__hapi": "^20.0.12",
-    "@types/hapi__inert": "^5.2.3",
-    "@types/hapi__vision": "^5.5.3",
-    "@types/javascript-state-machine": "^2.4.3",
-    "@types/jest": "26.0.24",
-    "@types/lodash": "^4.14.182",
-    "@types/node": "^18.0.0",
+    "@redocly/cli": "^1.2.0",
+    "@types/convict": "^6.1.4",
+    "@types/crypto-js": "^4.1.2",
+    "@types/hapi": "^18.0.11",
+    "@types/hapi__hapi": "^20.0.13",
+    "@types/hapi__inert": "^5.2.7",
+    "@types/hapi__vision": "^5.5.5",
+    "@types/javascript-state-machine": "^2.4.4",
+    "@types/jest": "29.5.5",
+    "@types/lodash": "^4.14.198",
+    "@types/node": "^20.6.3",
     "@types/promise-timeout": "^1.3.0",
     "@types/redis": "2.8.32",
     "@types/redis-mock": "^0.17.1",
-    "@types/ws": "^8.5.3",
-    "@typescript-eslint/eslint-plugin": "^5.29.0",
-    "@typescript-eslint/parser": "^5.29.0",
+    "@types/ws": "^8.5.5",
+    "@typescript-eslint/eslint-plugin": "^6.7.2",
+    "@typescript-eslint/parser": "^6.7.2",
     "add": "^2.0.6",
-    "eslint": "8.18.0",
-    "eslint-config-prettier": "^8.5.0",
-    "eslint-config-standard": "17.0.0",
-    "eslint-import-resolver-typescript": "^2.7.1",
+    "audit-ci": "^6.6.1",
+    "eslint": "8.49.0",
+    "eslint-config-prettier": "^9.0.0",
+    "eslint-config-standard": "17.1.0",
+    "eslint-import-resolver-typescript": "^3.6.0",
     "eslint-plugin-cucumber": "^2.0.0",
-    "eslint-plugin-import": "2.26.0",
+    "eslint-plugin-import": "2.28.1",
     "eslint-plugin-node": "11.1.0",
-    "eslint-plugin-prettier": "^4.0.0",
-    "eslint-plugin-promise": "6.0.0",
+    "eslint-plugin-prettier": "^5.0.0",
+    "eslint-plugin-promise": "6.1.1",
     "eslint-plugin-standard": "4.1.0",
-    "husky": "^5.2.0",
-    "jest": "^26.6.3",
+    "husky": "^8.0.3",
```

Comment:
Husky is updated but the git hooks are no longer being executed

Reply:
it does trigger for me. let me verify

Reply:
You can check for similar update here https://github.com/mojaloop/ml-testing-toolkit-shared-lib/pull/13